wire-ios-data-model/Source/MLS/SafeCoreCrypto.swift

```diff
@@ -69,7 +42,7 @@ public class SafeCoreCrypto: SafeCoreCryptoProtocol {
         let coreCrypto = try await coreCryptoDeferredInit(
             path: path,
             key: key,
-            ciphersuites: [CiphersuiteName.default.rawValue], nbKeyPackage: nil
+            ciphersuites: [1], nbKeyPackage: nil
```

Comment:
any chance CC remove the ciphersuites here if that's not used ?

Reply:
Yes, of course. I'll check that we have ticket for this.

Reply:
Added a ticket reference